VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -189,7 +190,8 @@ public Properties trackVideoSeek(String videoId,
             skipInterval = formatDoubleValue(skipInterval, 3);
             aEvent.data.putValue(Keys.OLD_TIME, oldTime);
             aEvent.data.putValue(Keys.NEW_TIME, newTime);
-            aEvent.data.putValue(Keys.SEEK_TYPE, Values.SKIP);
+            if (skipSeek){ aEvent.data.putValue(Keys.SEEK_TYPE, Values.SKIP);}
```

Comment:
So I made this a boolean because of the way the code handled the skip listener. It just passes a boolean to the seek function that states if it was the skip button or not. I guess it would be better to change this because it would not work in the case we have a forward button.